### ISSUE TITLE
BTS Protruding Pedestal Button unstyled

### ISSUE THREAD

**Issue by Luke18033** [BEEmod/BEE2-items#1464]
![20170122183337_1](https://cloud.githubusercontent.com/assets/19228201/22189495/b6ad3cc4-e0d1-11e6-91ae-8620492c86ee.jpg)

**Comment by ZombiePwner1** [comment]
They actually are styled they have concrete textures on their base.

**Comment by Luke18033** [comment]
No, they're not. Brush Templates are used for the base, which have their textures changed automatically by style. Look at the instances and you'll see it's not there.

**Comment by Luke18033** [comment]
And even if it were styled, it's not done very well and I would change the title, not close it.

**Comment by LautaroL20** [comment]
And here are the Styled BTS Protruding Pedestal, for your pleasure >:D
![20170417133358_1](https://cloud.githubusercontent.com/assets/12244376/25095558/98a27cc4-2372-11e7-9eeb-374d4e3d10b9.jpg)
![20170417133345_1](https://cloud.githubusercontent.com/assets/12244376/25095561/9e7eb5cc-2372-11e7-8869-2e3068b95886.jpg)
